themes/src/main/resources-community/theme/base/login/messages/messages_de.properties

```diff
@@ -379,4 +380,12 @@ loggingOutImmediately=Sofortige Abmeldung
 accountUnusable=Eine sp\u00E4tere Nutzung der Anwendung ist mit diesem Konto nicht mehr m\u00F6glich
 userDeletedSuccessfully=Nutzer erfolgreich gel\u00F6scht
 
+access-denied=Zugriff verboten
```

Comment:
Could you please change this to: "Zugriff verweigert"? instead of "verboten"? More common.

Reply:
Hups, makes sense, changed

Reply:
Thanks. 👍 @abstractj could you please trigger the CI run for this one, then? thanks.